core/README.md

```
@@ -22,6 +22,7 @@
   - [7.1. Manual post-deployment system configuration](#71-manual-post-deployment-system-configuration)
   - [7.2. Exporting the Keycloak Tazama realm](#72-exporting-the-keycloak-tazama-realm)
   - [7.3. Docker Compose YAML structure](#73-docker-compose-yaml-structure)
+  - [7.4. Adding tenants and users to Keycloak](#74-adding-tenants-and-users-to-keycloak)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)